libs/remix-solidity/src/compiler/types.ts

```diff
@@ -143,6 +143,7 @@ export interface Source {
 
 export interface CompilerInputOptions {
     optimize: boolean | number,
+    runs: number,
```

Comment:
Can it be any number or a subset of number ?
Let's take an example, if you want to display the status of a request you might want to use number literal :
```typescript
status: 200 | 400 | 500 | ...;
```
I'm not sure if it's the same in your case though.

Reply:
Usually It is 200. Further user can define it from 1 to bigger value

Reply:
ok, then keep it at number

Reply:
do you know the range if any? could it be 10000000? @Aniket-Engg